### ISSUE TITLE
Multiple download

### ISSUE THREAD

**Issue by burhanaksendir** [cyansoft-dev/simple_downloader#4]
Thanks. Great! 
Can you give an example for multiple file downloads?

**Comment by cyansoft-dev** [comment]
actually this plugin is only suitable for use on one widget, but if you want to use it for multiple files, you can use a custom statefull widget which initializes each file download.

**Comment by burhanaksendir** [comment]
Thanks. While the file is being downloaded, if the user exits the application, can the download continue from where it left off when the application is opened again?

**Comment by burhanaksendir** [comment]
"resuming" download would be nicer. Now I have to start from 0% everytime...

**Comment by ljsharp** [comment]
> "resuming" download would be nicer. Now I have to start from 0% everytime...

Yes it would be nicer. Though I like how this package works. Good job. I hope you can resolve it soon.
